### PR TITLE
Add beta CI

### PR DESCRIPTION
**Pull request by bidoubiwa**
Add a CI to publish and tests new features that might be implemented

### CODE REVIEW

File: .github/workflows/beta-tests.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  pull_request:
+    branches: ['!bump-meilisearch-v*.*.*-beta', '**-beta']
+
+jobs:
+  integration_tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # Strapi is not yet compatible with node 16
+        node: ['14', '16']
```
Comment (alallema) [inline .github/workflows/beta-tests.yml:17]: 16 or 18? Why there is 16 in the matrix if it didn't work?

**Review by alallema — COMMENTED** [review]
Good job!

File: .github/workflows/beta-tests.yml (hunk 1 above)
Comment (bidoubiwa) [reply .github/workflows/beta-tests.yml:17]: `16` works but `18` does not!

Comment (alallema) [reply .github/workflows/beta-tests.yml:17]: Yes so maybe the sentence should be:
` # Strapi is not yet compatible with node 18` no?

Comment (bidoubiwa) [reply .github/workflows/beta-tests.yml:17]: Oooh haha yes! 
```suggestion
        # Strapi is not yet compatible with node 18
        node: ['14', '16']
```

**Review by alallema — APPROVED** [review]
LGTM! 🍖

**Comment by bidoubiwa** [comment]
bors merge

**Pull request merged**